models/cart-discount/src/cart-discount-draft/presets/sample-data-fashion/employee-sale.ts

```diff
@@ -0,0 +1,33 @@
+import * as CartDiscountValueRelative from '@commercetools-test-data/cart-discount-value-relative';
+import { LocalizedString } from '@commercetools-test-data/commons';
+import {
+  CustomerGroupDraft,
+  TCustomerGroupDraft,
+} from '@commercetools-test-data/customer-group';
+import * as CartDiscountDraft from '../..';
+import { TCartDiscountDraftBuilder } from '../../../types';
+
+const customerGroupDraft = CustomerGroupDraft.presets.sampleDataFashion
+  .employee()
+  .build<TCustomerGroupDraft>();
+
+const employeeSale = (): TCartDiscountDraftBuilder =>
+  CartDiscountDraft.presets
+    .empty()
+    .value(
+      CartDiscountValueRelative.CartDiscountValueRelativeDraft.random()
+        .type('relative')
+        .permyriad(1500)
+    )
+    .cartPredicate(`customer.customerGroup.key = "${customerGroupDraft.key}"`)
+    // TODO: create `target` model
+    .target(undefined)
+    .name(LocalizedString.presets.empty()['en-US']('Employee Sale'))
+    .description(LocalizedString.presets.empty()['en-US']('employee_sale'))
+    .stackingMode('Stacking')
```

Comment:
Don't we have a constant for this value?

Reply:
Yep! Fixed in 863e4ba717ae8a5363ca6f47484e12aec4a8953c